src/main/java/micdoodle8/mods/galacticraft/core/items/ItemUniversalWrench.java

```diff
@@ -50,44 +51,6 @@ public EnumRarity getRarity(ItemStack par1ItemStack)
         return ClientProxyCore.galacticraftItem;
     }
 
-    @RuntimeInterface(clazz = "buildcraft.api.tools.IToolWrench", modID = CompatibilityManager.modidBuildcraft)
```

Comment:
Why are these Buildcraft compatibility methods removed?   Does the new "facing" mechanism mean that the need to reference the Buildcraft API [interface](https://github.com/BuildCraft/BuildCraftAPI/blob/8.0.x-1.12.2/api/buildcraft/api/tools/IToolWrench.java) has completely gone?   I'm not an expert on the current Buildcraft gameplay but maybe there are some things that that Buildcraft requires a wrench for that go beyond simple block rotations?

Reply:
Oops. I initially attempted to go a totally different way of doing BC compatibility, (which didn't work), and I guess I forgot to copy over the old implementation. I will re-add the compatibility methods.